llpc/tool/amdllpc.cpp

```diff
@@ -248,6 +248,15 @@ cl::opt<bool> DumpDuplicatePipelines(
     "dump-duplicate-pipelines",
     cl::desc("If TRUE, duplicate pipelines will be dumped to a file with a numeric suffix attached"), cl::init(false));
 
+// -llpc_opt: Override the optimization level passed in to LGC with the given one.  This options is the same as the
+// `-opt` option in lgc.  The reason for the second option is to be able to test the LLPC API.
+cl::opt<CodeGenOpt::Level> LlpcOptLevel("llpc-opt", cl::desc("The optimization level for amdllpc to pass to LLPC:"),
```

Comment:
I think the lgc -opt option is not actually in lgc; it is in llvm's target support. So couldn't you piggy-back on that too and avoid adding a new and awkwardly-named option here? lgc/state/LgcContext.cpp has extern opt<CodeGenOpt::Level> OptLevel to access it.

Reply:
Oh, I see you said why you want the new option. Sorry.